app/models/incident.rb

```diff
@@ -120,44 +134,71 @@ def reviewed?
     staff_reviews.present?
   end
 
-  # from https://gist.github.com/janko-m/2b2cea3e8e21d9232fb9
-  def to_claims_sql(table:)
+  # Fields we can't provide are commented where they appear in the claims schema.
+  # Field we're not providing in the first stage are implemented but commented.
+  def claims_fields(table:)
     report = driver_incident_report
 
     fields = {
       incident: {
+        # FilePrefix, FileNum
```

Comment:
don't understand what's happening with these comments.

Reply:
Either:

1. Documenting the fields we cannot implement due to the data not being present in the incidents database; or
2. Documenting the future implementation of fields which are present in the incidents database, but which are not part of the initial set of fields which Liza has requested we map from incidents to claims at this stage of the integration process.